tgui/packages/tgui/interfaces/RapidPipeDispenser.tsx

```diff
@@ -38,59 +38,116 @@ const TOOLS = [
   },
 ];
 
+type Directions = {
+  north: BooleanLike;
+  south: BooleanLike;
+  east: BooleanLike;
+  west: BooleanLike;
+};
+
+type Colors = {
```

Comment:
are these being used as an enum? I'm confused what this is doing

Reply:
I guess they're for forbidding certain connections for smart pipe. You can allow or forbid north connection for instance.

Reply:
Do you want me to change this to a bitflag? I just wanted to make some tweaks to UI...

Reply:
Renamed to DirectionsAllowed

Reply:
that's totally fine. though a bitflag would be optimal, i think just a more descriptive name would be fine